deploy/src/Microsoft.Azure.IIoT.Deployment/Microsoft.Azure.IIoT.Deployment.csproj

```diff
@@ -40,6 +40,7 @@
     <PackageReference Include="Serilog.AspNetCore" Version="6.0.1" />
 
     <PackageReference Include="System.Private.Uri" Version="4.3.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
@marcschier, is this additional package reference needed here given that there are no code changes ?

Reply:
Yes, other dependencies pull in 10.0.0.1 which has security issues flagged by CG.